core/jobs/batch_jobs/exp_version_history_computation_job.py

```diff
@@ -341,6 +341,7 @@ def create_version_history_models(
                 states_vh_at_v1, metadata_vh_at_v1, committer_ids_at_v1
             )
             vh_model_at_v1.update_timestamps()
+            vh_model_at_v1.put()
```

Comment:
Why is this changed?

Reply:
Ooh, ok, thanks for explaining. Note that the previous code should not have been written that way -- as a general rule, "get" operations should never change state, especially the datastore.